src/main/java/com/gladguys/alucoapi/repositories/customs/impl/CustomExamGradeRepositoryImpl.java

```diff
@@ -0,0 +1,32 @@
+package com.gladguys.alucoapi.repositories.customs.impl;
+
+import com.gladguys.alucoapi.entities.ExamGrade;
+import com.gladguys.alucoapi.entities.dto.ExamGradeDTO;
+import com.gladguys.alucoapi.repositories.customs.CustomExamGradeRepository;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public class CustomExamGradeRepositoryImpl implements CustomExamGradeRepository {
+
+	private JdbcTemplate jdbcTemplate;
+
+	public CustomExamGradeRepositoryImpl(JdbcTemplate jdbcTemplate) {
+		this.jdbcTemplate = jdbcTemplate;
+	}
+
+	@Override
+	public List<ExamGradeDTO> getGradesByExamId(Long id) {
+
+		StringBuilder sql = new StringBuilder();
+		sql.append(	" SELECT s.id as studentId, s.name as studentName, eg.grade as grade from exam_grade eg \n" +
```

Comment:
Se for pra fazer assim, não seria melhor utilizar só uma variável do tipo `String `não ? Creio que o `StringBuilder `seja útil quando as querys precisam de várias validações, e consequentemente vários `appends`.